docs/guides/api-gateway/get-started.mdx

```diff
@@ -8,582 +8,350 @@ tags:
   - microservices
 ---
 
-import Auth0 from "./img/auth0-create-api.png";
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
 
-# Get started - Deliver and secure APIs in production
+You've developed a world-class API, and now you want to make it available online.
 
-You've developed a world-class API, and now you want to make it available for clients to access. This guide takes you from
-bringing your API online in one line to configuring an API gateway that simplifies routing, security, contract management,
-protocol translation, and more, providing a single point of entry for clients and services accessing your APIs.
+Aside from the challenges involved in any API gateway deployment, you're looking
+for a solution that allows you to:
 
-## Prerequisites
+- Consistently apply security and traffic management policy in one place
+- Provide a single pane of glass for observability
+- Work identically in every cloud or environment
```

Comment:
I _think_ I know what you mean by "work identically," but is there a more explicit way to phrase this, even if it'd be a little more wordy?